config/default.schema.json

```diff
@@ -899,25 +897,10 @@
 			"default": "",
 			"description": "SystemId for migration of users from system"
 		},
-		"FEATURE_COURSE_COPY": {
+		"FEATURE_COPY_SERVICE_ENABLED": {
```

Comment:
now that we have one global feature flag, we should consider using it wherever we have copy services :)